fdb-record-layer-core/src/main/java/com/apple/foundationdb/record/provider/foundationdb/OnlineIndexScrubber.java

```diff
@@ -872,7 +872,7 @@ public Builder setScrubbingPolicy(@Nonnull final ScrubbingPolicy.Builder scrubbi
          */
         public OnlineIndexScrubber build() {
             validate();
-            OnlineIndexer.Config conf = new OnlineIndexer.Config(limit, maxRetries, recordsPerSecond, progressLogIntervalMillis, increaseLimitAfter, maxWriteLimitBytes);
+            OnlineIndexer.Config conf = new OnlineIndexer.Config(limit, maxRetries, recordsPerSecond, progressLogIntervalMillis, increaseLimitAfter, maxWriteLimitBytes, 0);
```

Comment:
It might be good to define a constant somewhere so that callers can use a value that underscores the fact that they are asking for unlimited time à la `ExecuteProperties.UNLIMITED_TIME`.

Reply:
Used `OnlineIndexer.Config.UNLIMITED_TIME`.